common-service-core/src/main/java/org/zowe/apiml/config/ApplicationInfo.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.config;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
```

Comment:
```suggestion
@Builder
```

From this perspective, some new values will be added, it sounds reasonable to use a builder instead of the all-argument constructor.

Reply:
done

Reply:
done